style.css

```diff
@@ -0,0 +1,67 @@
+* {
+  padding: 0;
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.nav {
+  background-color: #3c3a39;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  color: #fff;
+  padding: 15px;
+}
```

Comment:
Good job on using flexbox to align the content of the nav. Just change it to `header` and you will be all set. Keep up the good work :muscle: